frontend/nuxt.config.js

```diff
@@ -23,14 +26,42 @@ export default {
 
     // Global page headers: https://go.nuxtjs.dev/config-head
     head: {
-        title: 'Auctions UI',
+        title: 'Unified Auctions UI',
         htmlAttrs: {
             lang: 'en',
         },
         meta: [
             { charset: 'utf-8' },
             { name: 'viewport', content: 'width=device-width, initial-scale=1' },
-            { hid: 'description', name: 'description', content: '' },
+            {
+                hid: 'description',
+                name: 'description',
+                content: SITE_DESCRIPTION,
+            },
+            { property: 'og:site_name', content: 'Unified Auctions UI' },
+            { hid: 'og:type', property: 'og:type', content: 'website' },
+            {
+                hid: 'og:url',
+                property: 'og:url',
+                content: process.env.PRODUCTION_DOMAIN,
+            },
+            {
+                hid: 'og:description',
+                property: 'og:description',
+                content: SITE_DESCRIPTION,
+            },
+            {
+                hid: 'og:image',
+                property: 'og:image',
+                content: '/preview.jpeg',
+            },
+            {
+                hid: 'twitter:card',
```

Comment:
> To avoid any duplication when used in child components, please give a unique identifier with the `hid` key to the meta description. This way `vue-meta` will know that it has to overwrite the default tag.
https://nuxtjs.org/docs/features/meta-tags-seo/

`hid` should be part of every (or none tags, since we don't use meta-tags on other pages)